.editorconfig

```diff
@@ -14,3 +14,8 @@ max_line_length = 100
 [{Makefile,**/Makefile,runtime/doc/*.txt}]
 indent_style = tab
 indent_size = 8
+[*.lua]
```

Comment:
If we add non-standard editorconfig options we should add a comment to this file explaining what they're for.

Reply:
Ok, I've added comments